### PR TITLE
Products

### PR DESCRIPTION
**Pull request by manish-GH**
Added Products page, Wishlist and Cart. 

For navigation use the temporary bottom navbar. (Since it's temporary I haven't added styles).
The script tag will be removed once I shift to react.

Please review the rest
Products: https://keen-wright-b8e584.netlify.app/screens/product/
Wishlist: https://keen-wright-b8e584.netlify.app/screens/wishlist/
Cart: https://keen-wright-b8e584.netlify.app/screens/cart/

### CODE REVIEW

File: screens/Product/index.html [hunk 1]
```diff
@@ -35,7 +35,321 @@
 					</a>
 				</div>
 			</nav>
-			<main class="main-container"></main>
+			<main class="main-container">
+				<div class="main-grid-container">
+					<div class="main-grid-sidebar-products">
+						<section class="main-sidebar">
```
Comment (Shivani-1524) [inline screens/Product/index.html:41]: could use some margin on the top of the sidebar

File: styles.css [hunk 2]
```diff
@@ -2,21 +2,24 @@
...
 ::-webkit-scrollbar {
 	width: 10px;
 }
 
 ::-webkit-scrollbar-thumb {
-	border: 1px solid rgba(0, 0, 0, 0);
+	border: 1px solid var(--color-border-scrollbar);
 	background-clip: padding-box;
 	border-radius: 9999px;
-	background-color: #aaaaaa;
```
Comment (Shivani-1524) [inline styles.css:15]: use color variables. also, why have you given such a high border radius?

Comment (manish-GH) [reply styles.css:15]: Oh, about this, well I was experimenting with the scrollbar last night and completely forgot about it. Will change this ASAP

**Review by mmanjuprasad — APPROVED** [review]
seems like links are broken in preveiw
image
please check

**Comment by mmanjuprasad** [comment]
Give some padding on top for MY CART headline and make it catchy for eyes
rest looks good

**Comment by manish-GH** [comment]
> seems like links are broken in preveiw image please check

Yeah, I haven't added the navigation for the links, as of now all the navigation is managed by the bottom navbar

**Review by Sahil25061999 — APPROVED** [review]
most part looks good just a suggestion in terms of ui add a bit of padding for the card.

**Comment by manish-GH** [comment]
> most part looks good just a suggestion in terms of ui add a bit of padding for the card.

Just to clarify, are you referring to the image part or text part, or both?

**Pull request merged**